packages/cms-lib/api/fileManager.js

```diff
@@ -19,6 +19,8 @@ async function uploadFile(accountId, src, dest) {
 
   if (directory && directory !== '.' && directory !== '/') {
     formData.folderPath = directory;
+  } else {
+    formData.folderPath = '/';
```

Comment:
Is this something that changed with the shift to v3? See this makes me wonder if we should drop `&& directory !== '/'` since that case will work as intended via line 21.

Reply:
I might have missed this when migrating to v3. That's a good point, dropped that extra condition
```